Test/Utils/NetAdapterInfo/Impl.ps1

```diff
@@ -11,20 +11,17 @@ function Read-RawRemoteContainerNetAdapterInformation {
 
         $RemoteCommand = {
             $GetIPAddress = { ($_ | Get-NetIPAddress -AddressFamily IPv4).IPAddress }
-            $Fields = 'ifIndex', 'ifName', 'Name', 'MacAddress', 'MtuSize', @{L='IPAddress'; E=$GetIPAddress}
+            $Fields = 'ifIndex', 'Name', 'MacAddress', 'MtuSize', @{L='IPAddress'; E=$GetIPAddress}, @{L='ifName'; E='filledlater'}
```

Comment:
I'd prefer `Add-Member` here, since that `filledlater` does not work as one could expect.

Reply:
Done